modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ResultsBar.js

```diff
@@ -87,7 +111,7 @@ const ResultsBar = ({
 							searchValue
 						)}
 						className="component-link tbar-link"
-						href={clearResultsURL}
+						onClick={(event) => handleClick(event)}
```

Comment:
I feel like this solution does not fix the root cause of the issue. It hides the issue, by changing the behavior. I think the new behavior would be acceptable, if there is no way to fix the root cause, but are we ready to surrender just yet?

The root cause seems to be that `disabled` prop is not always set, but [only after `rowToggled` event](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SelectionControls.js#L123-L127). Can we make is so that it is always set?

Reply:
Hey @markocikos ,

Thanks for the review, you were right and I just might changes the behaviour too hastly. I created a new PR with the solution handling the underlying issue: https://github.com/liferay-frontend/liferay-portal/pull/2995

I close this PR